Makefile

```diff
@@ -58,6 +59,19 @@ remote-garden-up:
 remote-garden-down:
 	@./hack/local-development/remote-garden/stop.sh $(REMOTE_GARDEN_LABEL)
 
+.PHONY: start-all
+start-all:
+	# start a screen session and open all servers in named tabs
+	echo "starting screen session with all servers"
+	@screen -ls $(DEV_SCREEN_NAME) || screen -AdmS $(DEV_SCREEN_NAME) -t tab0 bash
+	@screen -S $(DEV_SCREEN_NAME) -X screen -t apiserver bash -c "make --debug=j start-apiserver; exec bash"
```

Comment:
It would be nice to wait the API server to start and to be reachable, e.g. `kubectl get shoot` to not return errors, before to start the other components.

Reply:
done